### PR TITLE
Override rpc config on CLI

### PR DESCRIPTION
**Pull request by tallywiesenberg**
The CLI can now override the RPC url set in configs it also updates the chain id that pytelliot points to. tested on goerli and rinkeby.

### CODE REVIEW

File: src/telliot_feed_examples/cli.py [hunk 1]
```diff
@@ -62,9 +62,23 @@
     type=int,
     default=500000,
 )
+@click.option(
+    "--rpc-url",
+    "-rpc",
+    "override_rpc_url",
```
Comment (oraclown) [inline src/telliot_feed_examples/cli.py:68]: Maybe `rpc_url_override`?

File: tests/cli/test_cli.py [hunk 2]
```diff
@@ -51,4 +51,39 @@ def test_cmd_tip():
     assert expected in result.output
 
 
+def test_rpc_override():
+    """Test the CLI option to override the RPC url provided in configs"""
+    runner = CliRunner()
+    result = runner.invoke(
+        cli,
+        [
+            "-lid",
```
Comment (oraclown) [inline tests/cli/test_cli.py:60]: Might error if other PR gets merged first, just fyi

File: src/telliot_feed_examples/cli.py [hunk 3]
```diff
@@ -62,9 +62,23 @@
...
+    help="override the config RPC url. requires also overriding the chain ID",
```
Comment (oraclown) [inline src/telliot_feed_examples/cli.py:69]: I'd remove the `. requires also overriding...` part, as it does this automatically behind the scenes it looks like.

**Review by oraclown — CHANGES_REQUESTED** [review]
Suggested changes in earlier comments

File: src/telliot_feed_examples/cli.py (hunk 3 above)
Comment (tallywiesenberg) [reply src/telliot_feed_examples/cli.py:69]: I agree

**Pull request merged**